db/migrate/20230518045652_create_topics.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTopics < ActiveRecord::Migration[7.0]
+  def change
+    create_table :topics do |t|
+      t.string :label
+      t.string :icon
```

Comment:
since you are validating presence on the controller, you should also add null: false here at database level:
```suggestion
      t.string :label, null: false
      t.string :icon, null: false
```